apps/galaxy/src/services/utils/axios.ts

```diff
@@ -3,15 +3,36 @@ import axios from "axios";
 import { environment } from "../constants/environment";
 import { getToken } from "@src/utils/tokenService";
 
-const token = getToken() || "";
 const REQUEST_TIMEOUT = 2000;
 
+type config = {
+  url: string;
+};
+
+const urlWhiteList = ["/staff/signin", "/franchisee/signup", "/staff/verify"];
+
 const customAxios = axios.create({
   baseURL: environment.API_BASE_URL,
   timeout: REQUEST_TIMEOUT,
-  headers: {
-    Authorization: token,
-  },
 });
 
+customAxios.interceptors.request.use(
+  function (config) {
```

Comment:
Instead of casting `config` as type `Config`, you want to announce that type def in function parameter. `function(config: Config)`

Reply:
Hang on, isn't the type supposed to be inferred by `axios` here? 

https://axios-http.com/docs/interceptors